evm/p2p/ecies.py

```diff
@@ -0,0 +1,144 @@
+import os
+import struct
+from hashlib import sha256
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import hashes, hmac
+from cryptography.hazmat.primitives.asymmetric import ec
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+from cryptography.hazmat.primitives.constant_time import bytes_eq
+
+from eth_utils import force_bytes
+
+from evm.p2p.constants import (
+    pubkey_len,
+)
+from evm.p2p.exceptions import DecryptionError
+from evm.utils.numeric import (
+    big_endian_to_int,
+    int_to_big_endian,
+)
+from evm.utils.secp256k1 import (
+    decode_public_key,
+    private_key_to_public_key,
+)
+
+
+cipher = algorithms.AES
+mode = modes.CTR
+curve = ec.SECP256K1()
+# ECIES using AES256 and HMAC-SHA-256-32
+key_len = 32
```

Comment:
Thoughts on renaming this and the others above it to `KEY_LEN`? 

I find that casing helps distinguish top level module variables from vars in the local context.